features/cellular/framework/common/CellularList.h

```diff
@@ -88,6 +99,7 @@ template <class T> class CellularList
             delete temp;
             temp = _head;
         }
+        _tail=NULL;
```

Comment:
Should you add
_head = NULL

Reply:
NULL _head breaks the while loop, so it is NULL already